keyboards/zinc/config.h

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+//#ifndef CONFIG_H
+//#define CONFIG_H
+
+#include "config_common.h"
+
+// GCC include 'config.h" sequence in qmk_firmware/keyboards/zinc/
+//   -include keyboards/zinc/config.h
+//   -include keyboards/zinc/rev?/config.h
+//   -include keyboards/zinc/rev?/keymaps/MAPNAME/config.h
+//   XXXX.c
+
+#include <serial_config.h>
+
+// GCC include search path in qmk_firmare/keyboards/zinc/
+//    #include "..." search starts here:
+//    #include <...> search starts here:
+//     keyboards/zinc/rev?/keymaps/MAPNAME
+//     keyboards/zinc
+//     keyboards/zinc/rev?
+//     .
+//     ./tmk_core
+//     ......
+
+#ifdef USE_Link_Time_Optimization
+  // LTO has issues with macros (action_get_macro) and "functions" (fn_actions),
+  //  so just disable them
+  #define NO_ACTION_MACRO
+  #define NO_ACTION_FUNCTION
+
+  #define DISABLE_LEADER
```

Comment:
This is no longer used. 

Leader keys were converted to a feature, so if you don't have `LEADER_ENABLE = yes` set, then it will be disabled. 

```suggestion
```

Reply:
> You may want to seriously consider converting this to the "split common" code, here.
> 
> You don't appear to be doing anything differently from the split code, so you could remove the matrix.c, serial, i2c and split_util files, and replace `CUSTOM_MATRIX = yes` with `SPLIT_KEYBOARD = yes` instead.

Thanks for your review!
Can't use "split common" code cause Zinc is using PD2 for serial.

Reply:
I removed  #define DISABLE_LEADER.

Reply:
Zinc uses not TRRS cord but TRS cord, I mean Zinc hasn't any hardware line like "Extra Data" of let's Split and each pro micro handles own LEDs.
The serial codes must handle multi-transaction to sync RGB LEDs. 
That's another reason why I use Helix serial codes.

Reply:
See, that's the problem, the helix added new local code, rather than expand upon the split common code. 

I wish they'd expanded, instead.

From what I can tell, a lot of this stuff could be merged, without too much effort. Just a lot of work. 

If you want to take a shot at it, i've opened an issue for this here: https://github.com/qmk/qmk_firmware/issues/4254

Because, really, all of this code should be merged, IMO.

Reply:
> See, that's the problem, the helix added new local code, rather than expand upon the split common code.

I think that is an unfair condemnation.

quantum/split_common/ was newly established in July.
keyboards/helix was added in January.

When @MakotoKurauchi worked, there was no common code to extend.

Reply:
@mtei That was not meant to be a condemnation, at all.  and I'm sorry if it came out that way. 

Namely, it's frustrating to have multiple pieces of code doing the same thing. 

For instance, the crkbd is based directly on the helix board, but all the improvements that you've made are NOT carried over to the crkbd. 

And I'm sorry if I dot the timeline wrong! 

But I think that there needs to be an effort to refactor the split common code, so we can pull out the split code for the helix and other boards, since that will allow more people to use and benefit from the amazing work that you and others have done on the code.